dev-haskell/ghc-lib-parser-ex/ghc-lib-parser-ex-8.10.0.16.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.6.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Algorithms on GHC parse trees"
+HOMEPAGE="https://github.com/shayne-fletcher/ghc-lib-parser-ex#readme"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	>=dev-haskell/uniplate-1.5:=[profile?]
+	>=dev-lang/ghc-8.2.1:=
+	=dev-haskell/ghc-lib-parser-8.10*:=[profile?]
```

Comment:
I'd prefer hackport-generated `>=...-8.10 <8.11` style of ranged depends. Makes it easier to grep and amend later.

Reply:
@trofi I believe this is what hackport generated, though I will double-check.

Upstream (doesn't seem to)[https://github.com/shayne-fletcher/ghc-lib-parser-ex/blob/master/ghc-lib-parser-ex.cabal#L122] specify a specific version, so I figured that is why hackport chose this style.

Reply:
@trofi I can confirm that the `>=dev-lang/ghc-8.2.1:=` is directly out of hackport.

Is this a hackport bug?

If so, how would you like me to change this? i.e. what upper bound should I use?

Reply:
This commit appears to have been added in your hlint PR. I'm not sure why it's here as well.

Reply:
It's because I rebased on top of that branch while the PR was still being reviewed, b/c I needed the newer hlint to run some tests. I've rebased again so this should be gone now